BUILD

```diff
@@ -0,0 +1,6 @@
+# This unpublished material is proprietary to Vaticle.
+# All rights reserved. The methods and
+# techniques described herein are considered trade secrets
+# and/or confidential. Reproduction or distribution, in whole
+# or in part, is forbidden except by express written permission
+# of Vaticle.
```

Comment:
The main repository license is AGPL.

Reply:
Fixed in https://github.com/typedb/typedb-sample-data/pull/11/commits/21bacc6bc0c946c5aa09aa29827721e6dbf09f56